tutorials/roofit/rf408_RDataFrameToRooFit.C

```diff
@@ -86,17 +91,20 @@ void rf408_RDataFrameToRooFit()
   };
 
   // Then, we move it into an RDataFrame action:
-  auto rooDataHist = dd.Book<double, double>(std::move(rdhMaker), {"x", "y"});
-
+  ROOT::RDF::RResultPtr<RooDataHist> rooDataHistResult = df.Book<double, double>(std::move(rdhMaker), {"x", "y"});
 
 
   // Run it and inspect the results
   // -------------------------------
 
+  // Get values from the RResultPtr objects to trigger the event loop and get the
```

Comment:
I would also expand this comment a bit: 
"At this point all RDF actions were defined (namely, the `Book`operation), so we can get values from the RResultPtr objects, trigger the event loop and get the actual RooFit data objects."

Reply:
And same comments for the python version.

Reply:
Ok will do, thanks for the suggestions!